homework/App.js

```diff
@@ -44,24 +100,9 @@ class App {
    * repo and its contributors as HTML elements in the DOM.
    * @param {number} index The array index of the repository.
    */
-  async fetchContributorsAndRender(index) {
+  async fetchContributorsAndRender(data, container) {
     try {
```

Comment:
I don't think you need to put this in a try-catch block

Reply:
Yes, no longer need.